contracts/base/FallbackManager.sol

```diff
@@ -37,6 +37,10 @@ contract FallbackManager is SelfAuthorized {
     }
 
     // @notice Forwards all calls to the fallback handler if set. Returns 0 if no handler is set.
```

Comment:
not using the multiline comment here because it doesn't play well with solhint-disable thing